src/plugins/data_views/server/index_patterns_api_client.ts

```diff
@@ -41,7 +43,7 @@ export class IndexPatternsApiServer implements IDataViewsApiClient {
         metaFields,
         type,
         rollupIndex,
-        indexFilter,
+        indexFilter: getIndexFilterDsl({ indexFilter, excludedTiers: this.excludedTiers }),
```

Comment:
Would it affect this logic of which path to use?
https://github.com/elastic/kibana/blob/ed29b977bd9d531170f48bcbff8ecbc1f5076556/src/plugins/data_views/public/data_views/data_views_api_client.ts#L101-L102

Reply:
Since this is server side code, it calls field_caps directly - `src/plugins/data_views/server/fetcher/index_patterns_fetcher.ts`